docs/features/cli.md

```diff
@@ -57,7 +24,7 @@ dagfactory [OPTIONS]
 | `--version` |       | Show the installed version of DAG Factory and exit |
 | `--help`    | `-h`  | Show this message and exit                         |
 
-### Version
+#### Identify the CLI version
```

Comment:
Is it the CLI's version or the dag-factory version? I guess we won't maintain CLI versions?

Reply:
yes, happy for it to be as is now. We can address it later if it confuses someone.